src/Engine/RadiumEngine.hpp

```diff
@@ -198,6 +201,22 @@ class RA_ENGINE_API RadiumEngine
 
     /// For internal resources management in a filesystem
     std::string m_resourcesRootDir;
+
+    /// active fbo and viewport management
```

Comment:
Be more informational on documentation on inner classes, even if they are private.